src/navigation/context.py

```diff
@@ -34,6 +41,8 @@ class Environment:
 
     ctx: Context
     NO_FIDUCIAL: ClassVar[int] = -1
+    post1: Post
```

Comment:
these should be part of `Environment`

Reply:
I changed how this works so not relevant anymore